### DISCUSSION TITLE
Run asciidoctor-web-pdf from Electron Main Process' NodeJS code

### DISCUSSION THREAD

**Discussion by jayczech23** [Mogztter/asciidoctor-web-pdf#580] [Q&A]
Hello,

I am trying to execute asciidoctor-web-pdf functionality inside of an electron app. I want to do this directly from code if possible, I have a few ideas on how to achieve this but wanted to see if there is a go-to / straightforward way.

**Comment by ggrossetie** [comment]
Hello!

That's a good question but since I've never tried to do it I can't really say...
The main difficulty will probably to replace Puppeteer (which runs an headless Chrome) by the Chromium instance provided by Electron. But again, I don't know if it's possible or if you want to do that...

Anyway, I'd be very interested in hearing from you if you manage to get it working.
